pinot-core/src/main/java/com/linkedin/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -120,6 +120,9 @@ public boolean isFinal() {
   }
   private static final Logger LOGGER = LoggerFactory.getLogger(LLRealtimeSegmentDataManager.class);
   private static final int KAFKA_MAX_FETCH_TIME_MILLIS = 1000;
+  private static final long ONE_MINUTE_MS = 60 * 1000L;
+  private static final long ONE_HOUR_MS = 60 * 60 * 1000L;
+  private static final int HUNDRED_K = 100000;
```

Comment:
I would just name this something like message count log threshold or something instead of hundred_k. The same thing applies to one minute/hour in millis.

If you want to express the duration in a human readable fashion, you can also do something like `HOURS.toMillis(1)` from `java.util.concurrent.TimeUnit`. I would only make it a constant if it's something that we might want to change.